# UCS Virtual Machine Manager Daemon
#
# Copyright 2011-2013 Univention GmbH
#
# http://www.univention.de/
#
# All rights reserved.
#
# The source code of the software contained in this package
# as well as the source package itself are made available
# under the terms of the GNU Affero General Public License version 3
# (GNU AGPL V3) as published by the Free Software Foundation.
#
# Binary versions of this package provided by Univention to you as
# well as other copyrighted, protected or trademarked materials like
# Logos, graphics, fonts, specific documentations and configurations,
# cryptographic keys etc. are subject to a license agreement between
# you and Univention.
#
# In the case you use the software under the terms of the GNU AGPL V3,
# the program is provided in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
# GNU Affero General Public License for more details.
#
# You should have received a copy of the GNU Affero General Public
# License with the Debian GNU/Linux or Univention distribution in file
# /usr/share/common-licenses/AGPL-3; if not, see
# <http://www.gnu.org/licenses/>.
#
# Andreas Büsching <buesching@univention.de>, 2011
#
msgid ""
msgstr ""
"Project-Id-Version: univention-management-console-module-uvmm 1.0.103-1\n"
"Report-Msgid-Bugs-To: packages@univention.de\n"
"POT-Creation-Date: 2011-11-25 20:38+0100\n"
"PO-Revision-Date: 2012-02-15 16:09+0100\n"
"Last-Translator: Univention GmbH <packages@univention.de>\n"
"Language-Team: German <de@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: umc/python/uvmm/storages.py:176
#, python-format
msgid "Could not retrieve details for domain %s"
msgstr "Die Details zur virtuellen Instanz %s konnten nicht abgerufen werden"

#: umc/python/uvmm/storages.py:166
msgid "Failed to check if the drive is used by any other virtual instance"
msgstr ""
"Die Prüfung ob das Laufwerk von einer anderen virtuellen Instanz verwendet "
"wird ist fehlgeschlagen."

#: umc/python/uvmm/domains.py:333
#, python-format
msgid "Failed to retrieve details for the server %(nodeURI)s"
msgstr ""
"Das Abrufen der Details zum physikalischen Server %(nodeURI)s ist "
"fehlgeschlagen"

#: umc/python/uvmm/profiles.py:65
msgid "Full virtualization (KVM)"
msgstr "Voll-Virtualisierung (KVM)"

#: umc/python/uvmm/profiles.py:66
msgid "Full virtualization (XEN)"
msgstr "Voll-Virtualisierung (XEN)"

#: umc/python/uvmm/domains.py:471
#, python-format
msgid "Invalid domain state: %s"
msgstr "Ungültiger Domain Status: %s"

#: umc/python/uvmm/domains.py:221
#, python-format
msgid "No valid source for disk \"%s\" found"
msgstr "Es fehlt eine gültige Quelle für das Gerät \"%s\""

#: umc/python/uvmm/profiles.py:67
msgid "Paravirtualization (XEN)"
msgstr "Paravirtualisierung (XEN)"

#: umc/python/uvmm/__init__.py:115
#: umc/python/uvmm/nodes.py:78
msgid "Physical servers"
msgstr "Physikalische Server"

#: umc/python/uvmm/uvmmd.py:67
msgid "The given UVMM command is not known"
msgstr "Der gegebene UVMM Befehl ist unbekannt"

#: umc/python/uvmm/storages.py:140
msgid "The given pool could not be found or is no file pool"
msgstr ""
"Der angegebene Speicherbereich konnte nicht gefunden werden oder ist kein "
"Verzeichnis"

#: umc/python/uvmm/domains.py:335
#, python-format
msgid "Unknown physical server %(nodeURI)s"
msgstr "Unbekannter physikalischer Server %(nodeURI)s"

#: umc/python/uvmm/profiles.py:122
msgid "Unknown profile"
msgstr "Unbekanntes Profil"

#: umc/python/uvmm/domains.py:312
msgid "Unknown profile given"
msgstr "Unbekanntes profil wurde angegeben"

#: umc/python/uvmm/__init__.py:117
msgid "Unknown query type"
msgstr "Unbekannter Anfragetyp"

#~ msgid "Could not find the drive"
#~ msgstr "Das Laufwerk konnte nicht gefunden werden"
